packages/dev-kit-italia/elements.ts

```diff
@@ -2,6 +2,8 @@
 import '@italia/accordion/it-accordion.js';
 import '@italia/accordion/it-accordion-item.js';
 import '@italia/button/it-button.js';
+import '@italia/callout/it-callout.js';
+import '@italia/callout/it-callout-more.js';
```

Comment:
```suggestion
import '@italia/callout/it-callout-more.js';
import '@italia/callout/it-callout-more-collapse.js';
```

Questo perché nel commento sopra in it-callout-more ho consigliato di rimuovere l'import.